lib/miq_expression.rb

```diff
@@ -645,6 +649,7 @@ def field_from_virtual_reflection?(field)
   end
 
   def attribute_supported_by_sql?(field)
+    return false if col_details[field].nil?
```

Comment:
@kbrock you could do that, but then you'd have a predicate which returns `nil`, which may be surprising